docs/resources/group_member.md

```diff
@@ -5,7 +5,7 @@ subcategory: "Security"
 
 This resource allows you to attach [users](user.md), [service_principal](service_principal.md), and [groups](group.md) as group members.
 
-To attach members to groups in the Databricks account, the provider must be configured with `host = "https://accounts.cloud.databricks.com"` on AWS deployments or `host = "https://accounts.azuredatabricks.net"` and authenticate using AAD tokens on Azure deployments
+To attach members to groups in the Databricks account, the provider must be configured with `host = "https://accounts.cloud.databricks.com"` on AWS deployments or `host = "https://accounts.azuredatabricks.net"` and authenticate using [AAD tokens](https://learn.microsoft.com/en-us/azure/databricks/dev-tools/api/latest/aad/) on Azure deployments
```

Comment:
@nkvuong rephrase to include https://registry.terraform.io/providers/databricks/databricks/latest/docs#special-configurations-for-azure instead. "AAD tokens" are ephemeral.